client-data/js/board.js

```diff
@@ -593,7 +597,20 @@ if (!Math.hypot) {
 		return Math.sqrt(x * x + y * y);
 	}
 }
-
+if (!Element.prototype.matches) {
+	Element.prototype.matches =
+		Element.prototype.matchesSelector ||
+		Element.prototype.mozMatchesSelector ||
+		Element.prototype.msMatchesSelector ||
+		Element.prototype.oMatchesSelector ||
+		Element.prototype.webkitMatchesSelector ||
+		function(s) {
+			var matches = (this.document || this.ownerDocument).querySelectorAll(s),
+				i = matches.length;
+			while (--i >= 0 && matches.item(i) !== this) {}
+			return i > -1;
+		};
+}
```

Comment:
Maybe this could be handled by the polyfill library too ? (see above)

Reply:
same as above then :+1:

Reply:
done